<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserViewWindow</class>
 <widget class="QMainWindow" name="UserViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User View</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#333</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="department_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black; font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Department: </string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black; font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="clock_in_out_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black; border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Clock in/out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none; font-size: 15px; color: white; font-weight: 700;</string>
    </property>
    <property name="text">
     <string>Currently: NOT CLOCKED IN</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="past_week_time_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>541</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="current_week_time_table">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> font-size: 20px; text-align: center; color: white;</string>
    </property>
    <property name="text">
     <string>Past Week</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> font-size: 20px; text-align: center; color: white;</string>
    </property>
    <property name="text">
     <string>Current Week</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black; font-size: 20px; border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_view_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black; font-size: 16px; border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Admin View</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
